packages/rune-games-cli/src/lib/cli.ts

```diff
@@ -28,6 +28,18 @@ export const cli = meow(
         type: "boolean",
         alias: "v",
       },
+      release: {
```

Comment:
I think we should list the flags inside --help command, otherwise the devs won't know that they can use these flags.

Reply:
added the usage under  Upload Options